lnbits/core/models.py

```diff
@@ -169,6 +169,8 @@ async def update_status(
     async def set_pending(self, pending: bool) -> None:
         from .crud import update_payment_status
 
+        self.pending = pending
```

Comment:
the function previously only updated the db, now updates the pending attribute aswell (see comment later)